db/seeds.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
+
 # This file should ensure the existence of records required to run the application in every environment (production,
```

Comment:
dont leave commented code

Reply:
Rubocop tells us that this comment must be at the beginning of each file. :c